ANSWERS.md

```diff
@@ -0,0 +1,8 @@
+1.  What are PropTypes used for?
+  PropTypes are used to validate the properties coming into a component. The properties would need to be the correct type to pass the test.
+1.  What is a lifecycle event in React?
+  A lifecycle event is any change that takes a place within the app. For example, after the app is initially rendered the componentDidMount method will run. If any state has changed in this method it will be passed to the app and changes will be rerendered to the screen.
```

Comment:
Think of it more on the component level, lifecycle event is more about a components Mounting, Updating, and Unmounting stages.